ARDemos/ARDemos/BottomBar.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+class BottomBar: UIView {
+
+    private let scrollView = UIScrollView()
+    private var models: [Model]?
+    
+    var onTap: ((String) -> Void)?
```

Comment:
I like the fact that you are using composition with callbacks instead of delegations. 
You can use typealias to make the code slighty more readable

Reply:
done 👍 1d01e93